### ISSUE TITLE
[ViewSettingsDialog]: [Color contrast not upto the standard in few themes]

### ISSUE THREAD

**Issue by Pranjal-98** [SAP/ui5-webcomponents#12132]
### Describe the bug

In the ViewSettingsDialog component, the Reset button’s color fails to meet the minimum required color contrast ratio under certain themes. The color is defined by the component itself and this issue is occuring in our custom themes. 

In the below added screenshot, you can see the Rest button is not visible clearly because of similar color of the button and background behind. 

<img width="1728" height="818" alt="Image" src="https://github.com/user-attachments/assets/767e96f5-6188-4d7e-a102-f4cf041917ac" />

### Expected Behaviour

The Reset button should be clearly visible in all the themes and minimum color contrast should be fulfilled.

### Screenshots or Videos

https://github.com/user-attachments/assets/fd0601bb-04f3-4970-a66f-d4fa4688a40a

### UI5 Web Components for React Version

v2.10.0

### UI5 Web Components Version

v2.10.0

### Browser

Chrome

### Relevant log output

```shell

```

### Organization

SAP Labs

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.

**Comment by s-todorova** [comment]
Hello @SAP/ui5-webcomponents-topic-b,

Since the 'Reset' button is part of the ViewSettingsDialog's shadow dom, it is not customizable and themeable by standard overriding of `var(--sapButton_Lite_TextColor); `.

Could you please take a look at this, maybe consider exposing a css part for this button?
Unsure if I should treat this as a bug or a feature request, since component works correctly in official themes but the limitation introduces contrast issue in custom ones.

Best,
Siyana

**Comment by nnaydenow** [comment]
Hi @Pranjal-98,

Could you please share how do you customize the used theme? Is it by UI Theme Designer or component by component?

Regards,
Nayden

**Comment by Pranjal-98** [comment]
Hi @nnaydenow,
It is by UI Theme Designer.

Regards,
Pranjal

**Comment by Pranjal-98** [comment]
Hi @nnaydenow ,
Did you get a chance to have a look at the issue?

Regards,
Pranjal

**Comment by nnaydenow** [comment]
Hi @Pranjal-98,

Thanks for sharing the details. I had a chance to review the issue, and based on the screenshots, it looks like the problem originates from the `sapPageHeader_Background` theming parameter being changed.

Since this is part of a custom theme, we unfortunately can’t guarantee that all aspects (such as contrast ratio or cross-component behavior) will work as expected. In such cases, it’s the responsibility of the theme creator is to ensure the parameters produce a consistent and compatible theme.

Keep in mind that theming parameters often apply to groups of components, so adjusting one can unintentionally impact multiple places in the UI.

To illustrate this, here’s an example showing how modifying this parameter can lead to inconsistencies across different components: [Example](https://sap.github.io/ui5-webcomponents/play/#eyJpbmRleC5odG1sIjp7Im5hbWUiOiJpbmRleC5odG1sIiwiY29udGVudCI6IjwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuPCFET0NUWVBFIGh0bWw-XG48aHRtbCBsYW5nPVwiZW5cIj5cblxuXG5cblxuXG48aGVhZD5cbiAgICBcbiAgICA8c3R5bGU-XG4gICAgICAqOm5vdCg6ZGVmaW5lZCkge1xuICAgICAgICBkaXNwbGF5OiBub25lO1xuICAgICAgfVxuXG4gICAgaHRtbCB7XG4gICAgICBmb3JjZWQtY29sb3ItYWRqdXN0OiBub25lO1xuICAgIH1cbiAgICA8L3N0eWxlPlxuXG4gICAgXG4gICAgPHN0eWxlPlxuICAgICAgKjpub3QoOmRlZmluZWQpIHtcbiAgICAgICAgZGlzcGxheTogbm9uZTtcbiAgICAgIH1cblxuICAgIGh0bWwge1xuICAgICAgZm9yY2VkLWNvbG9yLWFkanVzdDogbm9uZTtcbiAgICB9XG4gICAgPC9zdHlsZT5cblxuICAgIFxuICAgIDxzdHlsZT5cbiAgICAgICo6bm90KDpkZWZpbmVkKSB7XG4gICAgICAgIGRpc3BsYXk6IG5vbmU7XG4gICAgICB9XG5cbiAgICBodG1sIHtcbiAgICAgIGZvcmNlZC1jb2xvci1hZGp1c3Q6IG5vbmU7XG4gICAgfVxuICAgIDwvc3R5bGU-XG5cbiAgICBcbiAgICA8c3R5bGU-XG4gICAgICAqOm5vdCg6ZGVmaW5lZCkge1xuICAgICAgICBkaXNwbGF5OiBub25lO1xuICAgICAgfVxuXG4gICAgaHRtbCB7XG4gICAgICBmb3JjZWQtY29sb3ItYWRqdXN0OiBub25lO1xuICAgIH1cbiAgICA8L3N0eWxlPlxuXG4gICAgPG1ldGEgY2hhcnNldD1cIlVURi04XCI-XG4gICAgPG1ldGEgbmFtZT1cInZpZXdwb3J0XCIgY29udGVudD1cIndpZHRoPWRldmljZS13aWR0aCwgaW5pdGlhbC1zY2FsZT0xLjBcIj5cbiAgICA8dGl0bGU-U2FtcGxlPC90aXRsZT5cbiAgICA8bGluayByZWw9XCJzdHlsZXNoZWV0XCIgaHJlZj1cIi4vbWFpbi5jc3NcIj5cbjwvaGVhZD5cblxuPGJvZHkgc3R5bGU9XCJiYWNrZ3JvdW5kLWNvbG9yOiB2YXIoLS1zYXBCYWNrZ3JvdW5kQ29sb3IpOyBoZWlnaHQ6IDQ1MHB4O1wiPlxuICAgIDwhLS0gcGxheWdyb3VuZC1mb2xkLWVuZCAtLT5cblxuXHRNdWx0aUNvbWJvQm94OlxuXHQ8YnIgLz5cblx0PHVpNS1tdWx0aS1jb21ib2JveCBzaG93LXNlbGVjdC1hbGwgb3Blbj5cblx0XHQ8dWk1LW1jYi1pdGVtIHRleHQ9XCJhc2RzXCI-PC91aTUtbWNiLWl0ZW0-XG5cdDwvdWk1LW11bHRpLWNvbWJvYm94PlxuXHQ8YnIgLz5cblx0PGJyIC8-XG5cdDxiciAvPlxuXHQ8YnIgLz5cblx0PGJyIC8-XG5cdDxiciAvPlxuXHQ8YnIgLz5cblx0UGFnZTpcblx0PGJyIC8-XG5cdDx1aTUtcGFnZSBpZD1cInBhZ2VcIiBiYWNrZ3JvdW5kLWRlc2lnbj1cIkxpc3RcIiBzaG93LWZvb3Rlcj5cblx0XHQ8dWk1LWJhciBpZD1cImhlYWRlclwiIGRlc2lnbj1cIkhlYWRlclwiIHNsb3Q9XCJoZWFkZXJcIj5cblx0XHRcdDx1aTUtYnV0dG9uIGljb249XCJob21lXCIgdG9vbHRpcD1cIkdvIGhvbWVcIiBzbG90PVwiZW5kQ29udGVudFwiIGRlc2lnbj1cIlRyYW5zcGFyZW50XCI-PC91aTUtYnV0dG9uPlxuXHRcdDwvdWk1LWJhcj5cblx0XHQ8c3Bhbj5cblx0XHRcdFNvbWUgcGFnZSdzIGNvbnRlbnRcblx0XHQ8L3NwYW4-XG5cdDwvdWk1LXBhZ2U-XG5cdDx1aTUtZGlhbG9nIGhlYWRlci10ZXh0PVwiVGVzdFwiIG9wZW4gc3R5bGU9XCJtYXJnaW4tdG9wOiAzNjBweDtcIj5EaWFsb2c8L3VpNS1kaWFsb2c-XG5cdDx1aTUtdmlldy1zZXR0aW5ncy1kaWFsb2cgb3Blbj5cblx0XHQ8dWk1LXNvcnQtaXRlbSBzbG90PVwic29ydEl0ZW1zXCIgdGV4dD1cIk5hbWVcIj48L3VpNS1zb3J0LWl0ZW0-XG5cdDwvdWk1LXZpZXctc2V0dGluZ3MtZGlhbG9nPlxuICAgIDwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuICAgIDxzY3JpcHQgdHlwZT1cIm1vZHVsZVwiIHNyYz1cIm1haW4uanNcIj48L3NjcmlwdD5cbjwvYm9keT5cblxuPC9odG1sPlxuPCEtLSBwbGF5Z3JvdW5kLWZvbGQtZW5kIC0tPlxuIn0sIm1haW4uanMiOnsibmFtZSI6Im1haW4uanMiLCJjb250ZW50IjoiaW1wb3J0IFwiLi9wbGF5Z3JvdW5kLXN1cHBvcnQuanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy9kaXN0L0J1dHRvbi5qc1wiO1xuXG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMtZmlvcmkvZGlzdC9WaWV3U2V0dGluZ3NEaWFsb2cuanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy1maW9yaS9kaXN0L1NvcnRJdGVtLmpzXCI7XG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMtZmlvcmkvZGlzdC9QYWdlLmpzXCI7XG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMvZGlzdC9CYXIuanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy9kaXN0L011bHRpQ29tYm9Cb3guanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy9kaXN0L011bHRpQ29tYm9Cb3hJdGVtLmpzXCI7In0sIm1haW4uY3NzIjp7Im5hbWUiOiJtYWluLmNzcyIsImNvbnRlbnQiOiJcdFx0OnJvb3Qge1xuXHRcdFx0LS1zYXBQYWdlSGVhZGVyX0JhY2tncm91bmQ6IGJsYWNrICFpbXBvcnRhbnQ7XG5cdFx0fVxuIn19)

<img width="800" height="704" alt="Image" src="https://github.com/user-attachments/assets/f4db93e2-3791-42b9-a128-aea8189b48c7" />

I recommend reviewing the [UI Theme Designer documentation](https://help.sap.com/docs/btp/ui-theme-designer/creating-new-theme?locale=en-US) to identify which parameters are safe to adjust and to test their impact across your app. Unfortunately, we can’t provide a direct fix for this scenario.

Best regards,
Nayden
